test_data/core/dns_base.zone

```diff
@@ -13,3 +13,12 @@ _services._dns-sd._udp	PTR	_nmos-registration._tcp
 _services._dns-sd._udp	PTR	_nmos-register._tcp
 _services._dns-sd._udp	PTR	_nmos-query._tcp
 _services._dns-sd._udp	PTR	_nmos-system._tcp
+_services._dns-sd._udp	PTR	_nmos-auth._tcp
```

Comment:
Did there ought to be an mDNS equivalent of this, or is testing with `DNS_SD_MODE = 'multicast'` and `ENABLE_AUTH = True` not supported?

Reply:
It is supported, but I'd envisage that an IS-10 compatible auth server would already be advertising via mDNS in that case so there would be no need to replicate it. This is only required because we have to assume the role of DNS server which would otherwise be handled externally.

Reply:
That makes sense. Thanks. I didn't end up adding this in the IS-07 MQTT testing PR, but maybe we should add documentation that explains if you're running a non-NMOS-aware auth server or MQTT broker, and using mDNS, then using `dns-sd -R` (with mDNSResponder/mdnsd) or `avahi-publish -s` (with Avahi utils installed) could work.